getImportantAnimals

```diff
@@ -0,0 +1,17 @@
+# get most talked about animals
+
```

Comment:
Small detail: the function comment should be directly above the function definition: can you remove the empty line between your comment and `def getImportantAnimals...`?